### DISCUSSION TITLE
Change the nether biome type

### DISCUSSION THREAD

**Discussion by TimDegraaf** [BG-Software-LLC/SuperiorSkyblock2#816] [Implemented Feature Requests] [answered]
Creating nether islands the biome is a plains type. I am suggesting we can change the type like we can in the  island type but for the nether. I believe players will like it.

**Comment by OmerBenGera** [comment]
> Creating nether islands the biome is a plains type. I am suggesting we can change the type like we can in the island type but for the nether. I believe players will like it.

The biome in the nether should be nether biome, and in the end it should be the end's biome, and not plains.
If thats not the case, open an issue.

**Reply by Skizzles** [reply]
> > Creating nether islands the biome is a plains type. I am suggesting we can change the type like we can in the island type but for the nether. I believe players will like it.
> 
> The biome in the nether should be nether biome, and in the end it should be the end's biome, and not plains. If thats not the case, open an issue.

I believe he means the ability to set biome in the Nether and End to different biomes that those dimensions hold. (Which would be pretty awesome)

**Comment by OmerBenGera** [comment, marked as answer]
Added in latest dev build
